snippets/curl/nb_jobs.sh

```diff
@@ -0,0 +1,80 @@
+!/bin/sh
+
```

Comment:
Minor: Can rename this script to get_nb_jobs.sh for consistency.

Reply:
done